doc/websocket.qbk

```diff
@@ -152,23 +159,23 @@ been upgraded. The HTTP Upgrade request must include the Host HTTP field,
 and the URI of the resource to request. `hanshake` is used to send the
 request with the required host and resource strings.
 ```
-    websocket::stream<ip::tcp::socket> ws(ios);
+    beast::websocket::stream<boost::asio::ip::tcp::socket> ws(ios);
     ...
-    ws.set_option(websocket::keep_alive(true));
+    ws.set_option(beast::websocket::keep_alive(true));
     ws.handshake("ws.mywebapp.com:80", "/cgi-bin/bitcoin-prices");
```

Comment:
Consider using the canonical `example.com` domain in your examples and documentation - see [RFC 2606](https://tools.ietf.org/html/rfc2606)).